.gitignore

```diff
@@ -101,7 +101,6 @@ ENV/
 
 #cdk 
 *cdk.out*
-*.d.ts
```

Comment:
you can also leave the ignore as is and use an exclude to only not ignore this file: 
```
*.d.ts
!source/infrastructure/global.d.ts
```

Reply:
valid, but I saw no reason to do this as the cdk output is no longer bundled under /source so this is, in theory, an incorrect exclusion under the "cdk" header